packages/mission-center/mission-center.pacscript

```diff
@@ -0,0 +1,60 @@
+pkgname="mission-center"
+pkgver="0.5.2"
+pkgdesc="Monitor your CPU, Memory, Disk, Network and GPU usage"
+arch=("amd64" "arm64")
+url="https://gitlab.com/mission-center-devs/mission-center"
+license=("GPL-3.0-or-later")
+depends=(
+  "dmidecode"
+  "libadwaita-1-0"
+)
+makedepends=(
+  "blueprint-compiler"
+  "cargo"
+  "meson"
+  "gettext"
+  "pkg-config"
+  "python3-pip"
+  "python3-gi"
+  "libdrm-dev"
+  "libudev-dev"
+  "libgbm-dev"
+  "libpango1.0-dev"
+  "libgdk-pixbuf-2.0-dev"
+  "libgtk-4-dev"
+  "libdbus-1-dev"
+  "libadwaita-1-dev"
+)
+checkdepends=("appstream-util")
+source=("${url}/-/archive/v${pkgver}/${pkgname}-v${pkgver}.tar.gz")
```

Comment:
I don't like that url is a variable here. It's not "obfuscated" but it doesn't feel right.

Reply:
Uhm.... it looks fine to me....
But eh, since you worry about that....

Reply:
[#634264c](https://github.com/pacstall/pacstall-programs/pull/6383/commits/634264cf7425bb6a37ff0adf483717e7b9031e5a)

Happy?